src/client/components/pages/search.tsx

```diff
@@ -157,6 +160,26 @@ class SearchPage extends React.Component<Props, State> {
 					searchResultsCallback={this.searchResultsCallback}
 					size={this.props.resultsPerPage}
 				/>
+				<div className="text-center">
+					{results.length === 0 &&
+					<div>
+						<hr className="thin"/>
+						<h2 style={{color: '#754e37'}}>
+						No results found
+						</h2>
+					</div>}
+
+					<Row>
+						{results.length === 0 &&
+							<small>Make sure the spelling is correct, and that
+								 you have selected the correct type in the search bar.
+							</small>}
+						<hr className="wide"/>
+						<h3>Are we missing an entry?</h3>
+						<CallToAction query={this.props.query}/>
```

Comment:
```suggestion
						<CallToAction query={query}/>
```

Just realized when I tried locally that we need to use the query from state and not props, otherwise changes won't be taken into account (search for 'abc' -> CallToAction set with query='abc' -> search for 'def' -> CallToAction still set to query='abc')